src/views/v2/month/calendar-body/day/multiday-events/multiday-event.php

```diff
@@ -24,93 +25,22 @@
  * @version TBD
  */
 
-use Tribe__Date_Utils as Dates;
+$classes = tribe_events_multiday_classes( $event, $day_date, $is_start_of_week, $today_date );
 
-/*
- * To keep the calendar accessible, in the context of a week, we'll print the event only on either its first day
- * or the first day of the week.
- */
-$should_display = in_array( $day_date, $event->displays_on, true )
```

Comment:
I think all this logic was fine to keep in this template, as it's very specific to this template and nowhere else.